src/components/collection_voting_slab/ResearchProposalVoteSlab.js

```diff
@@ -91,6 +92,27 @@ const animationAttributes = {
   transition:{duration: 0.5, ease: [0.50, 0.62, 0.23, 0.98]}
 };
 
+const highlightedWords = [
+  {
+    bgColor: 'rgba(0,0,100,.2)',
+    textColor: '#0948B7',
+    words: [
+      'Health',
+      'Medical',
+      'Biomedical',
+      'Disease',
+      'Methods',
+      'Algorithm',
+      'Population',
+      'Origin',
+      'Ancestry',
+      'Controls',
+      'Commercial',
+      'Profit'
+    ]
+  }
+];
```

Comment:
I'm torn on where to define this. I like having the color options local to the component, but I think the words themselves would be more of a fit in a service-level utility. For this PR, I think this is fine. If we add this feature to other components (I could see it in the Application Info component too), then we should pull this up to a higher level of abstraction.